torch/_inductor/decomposition.py

```diff
@@ -361,7 +357,7 @@ def pad_bmm(mat1, mat2, m_padded_length, k_padded_length, n_padded_length):
         mat1 = pad_dim(mat1, k_padded_length, 2)
         mat2 = pad_dim(mat2, k_padded_length, 1)
         return torch.ops.aten.bmm(mat1, mat2)
-    elif config.shape_padding_bmm and n_padded_length != 0:
+    elif n_padded_length != 0:
```

Comment:
Can you leave a comment here that you go through decomp multiple times if you have to pad multiple dimensions? `elif`s are a bit confusing

Reply:
Comment added